docs/modules/ROOT/pages/release-notes/release-notes-2021-1.adoc

```diff
@@ -0,0 +1,70 @@
+* [https://github.com/payara/Payara/pull/5040[FISH-505]] - Server instance tries to load Application not assigned to instance 
+* [https://github.com/payara/Payara/pull/4898[FISH-515]] - HTTP POST request returns 500 on Zulu JDK8 
+* [https://github.com/payara/patched-src-grizzly/pull/27[FISH-631]] - Infinite loop in Grizzly SSL handshake causing deadlock 
+* [https://github.com/payara/Payara/pull/5051[FISH-642]] - Application Logging not performed on Payara 5 Cluster instances 
+* [https://github.com/payara/Payara/pull/5009[FISH-652]] - Error in MP JWT validation when retrieving JWKS key from remote location 
+* [https://github.com/payara/Payara/pull/5029[FISH-743]] - HK2 Class Parsing ClassCastException 
+* [https://github.com/payara/Payara/pull/5070[FISH-744]] - When creating a Resource Adapter Config via Admin Console only thread pools from 'Server-Config' is listed under Thread Pool ID 
+* [https://github.com/payara/Payara/pull/4997[FISH-765]] - [Community - https://github.com/sgflt[sgflt]] WebModule doesn't respect virtual server configuration 
+* [https://github.com/payara/Payara/pull/5010[FISH-790]] - Refactor, fix bugs and make immutable JavaEEContextUtil 
+* [https://github.com/payara/Payara/pull/5012[FISH-796]] - Fix Clustered Singleton bugs / add tests 
+* [https://github.com/payara/Payara/pull/5086[FISH-876]] - Incorrect Hashicorp Vault MP Config source blocks deployment and usage of MP Config 
+* [https://github.com/payara/Payara/pull/5092[FISH-877]] - Cloud MP Config Sources ignore ordinal defined in Domain 
+* [https://github.com/payara/Payara/pull/5065[FISH-885]] - OpenAPI document creation failed when using @Schema annotation with Enum 
+* [https://github.com/payara/Payara/pull/5090[FISH-892]] - Hashicorp vault secret value not picked up by MicroProfile after restart domain 
+* [https://github.com/payara/Payara/pull/5098[FISH-990]] - OpenTracing Active Span is NULL when retrieved in EJB tracer on remote execution 
+* [https://github.com/payara/Payara/pull/5020[FISH-994]] - [Community - https://github.com/sgflt[sgflt]] Package jaxws opentracing to embedded Payara  
+* [https://github.com/payara/Payara/pull/5082[FISH-1007]] - [Community - https://github.com/sgflt[sgflt]] Payara logback libs produce NPE 
+* [https://github.com/payara/Payara/pull/5071[FISH-1015]] - [Community - https://github.com/avpinchuk[avpinchuk]] Add appropriate application name to GAV deployments 
+* [https://github.com/payara/Payara/pull/4441[FISH-1017]] - [Community - https://github.com/bjetal[bjetal]] Payara can close JarFile instances used by current URLClassLoaders 
+* [https://github.com/payara/Payara/pull/5102[FISH-1018]] - Class Loader leaks on redeploy 
+* [https://github.com/payara/Payara/pull/5101[FISH-1019]] - [Community - https://github.com/bhanuurivalled[bhanuunrivalled]] GlassFishProperties NPE when initialised with null properties
+
+== Component Upgrades
+
+* [https://github.com/payara/Payara/pull/5014[FISH-791]] - Upgrade to Hazelcast 4
```

Comment:
We should add a paragraph at the top that due to this upgrade of Hazelcast, rolling updates are not supported.